### PR TITLE
Code and documentation overhaul

### PR DESCRIPTION
**Pull request by eugeneli**
Hi, in this branch I did the following:

- Added support for devnet
- Removed Api calls that weren't working or deprecated
- Wrapped ark-js for secure creation and sending of transactions (!!!)
- Transactions sent are also broadcast to multiple nodes to ensure delivery
- Cleaned up the documentation
- Refactored code to be cleaner

### CODE REVIEW

File: src/transaction.js [hunk 1]
```diff
@@ -1,68 +1,99 @@
...
+                "version": "0.3.0",
+                "port": 1,
+                "nethash": Network.hash
+            }
+        };
+
+        Api.post(params, callback);
+
+        broadcastTransactions(params, Network.seeds);
+        broadcastTransactions(params, Network.peers);
```
Comment (fix) [inline src/transaction.js]: It is quite hard way here, broadcast to seeds should be enough.

File: src/init.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Init.initP;
+
+Init.init = (networkName) => {
+    Init.initP = new Promise((resolve, reject) => {
+        Network.useNet(networkName).then(() => {
+            Peer.getPeersList((err, success, response) => {
+                if(response != null && response.success)
+                {
+                    Network.peers = response.peers.filter((peer) => peer.status == "OK" && peer.delay <= 100);
```
Comment (fix) [inline src/init.js]: If the node is not well connected you get 0 peers. i would skip the delay checking for lite clients as it is not strategic (above all if tx are broadcasted)

**Comment by fix** [comment]
can you merge README.md and update the PR?

**Comment by eugeneli** [comment]
Made your changes and cleaned up some more stuff. It's good to go @fix 👍

**Pull request merged**